src/language/from_relay/mod.rs

```diff
@@ -1808,4 +1813,126 @@ def @main(%x: Tensor[(3), float32]) -> Tensor[(3), float32] {
 (compute softmax (access (compute softmax (access (access-tensor x) 0)) 0))
 "#
     );
+
+    #[bench]
+    fn mobilenet(b: &mut Bencher) {
+        let filename = PathBuf::from(format!(
+            "{}/models/mobilenet-shallow.relay",
+            env!("CARGO_MANIFEST_DIR")
+        ));
+        let relay = std::fs::read_to_string(&filename).unwrap();
+        let relay_str = relay.clone();
+
+        // The number of times to run each program and compare their
+        // outputs.
+        // TODO(@gussmith23) # random samples chosen arbitrarily
+        const SAMPLES: usize = 3;
+
+        // Random number generator for generating random tensors.
+        const SEED: u64 = 23;
+        let mut tensor_rng = SmallRng::seed_from_u64(SEED);
+
+        let module = tvm::ir::module::IRModule::parse("", relay);
+
+        let (expr, shapes_vec) = super::from_relay(&module);
+
+        // Run interpreters and compare output.
+        let script_filepath = format!(
+            "{}/src/language/from_relay/run_relay.py",
+            env!("CARGO_MANIFEST_DIR")
+        );
+        // https://www.reddit.com/r/rust/comments/38jhva/piping_string_to_child_process_stdin/crvlqcd/?utm_source=reddit&utm_medium=web2x&context=3
+        // Output filename
+        // TODO(@gussmith23) Do we want this RNG to use SEED?
+        // I initially attempted to do this, but was running into issues
+        // (I think the same filename kept being generated b/c I wasn't
+        // using the RNG carefully...but maybe there's also something
+        // wrong w/ how I'm reading files!)
+        let output_filepath = std::env::temp_dir().with_file_name(format!(
+            "output-{}.npy",
+            rand::thread_rng()
+                .sample_iter(&rand::distributions::Alphanumeric)
+                .take(30)
+                .collect::<String>()
+        ));
+
+        let mut cmd = Command::new("python3");
+        cmd.arg(script_filepath);
+        cmd.arg(&output_filepath);
+        cmd.stdin(std::process::Stdio::piped())
+            .stdout(std::process::Stdio::piped())
+            .stderr(std::process::Stdio::piped());
+        let mut env = HashMap::default();
+        let mut index = 0;
+        for (name, shape) in shapes_vec.iter() {
+            // TODO(@gussmith23) output type assumption
+            let value = if index < 1 {
+                index += 1;
```

Comment:
```suggestion
        for (index, (name, shape)) in shapes_vec.iter().enumerate() {
            // TODO(@gussmith23) output type assumption
            let value = if index < 1 {
```
I'm not sure if this is your code or my code, but something like this should work too and avoids needing to increment the index variable.